cmd/policy/main.go

```diff
@@ -0,0 +1,397 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"go.uber.org/zap"
+
+	"github.com/veraison/common"
+	"github.com/veraison/policy"
+)
+
+func runCommand(config *common.Config, command string, args []string, logger *zap.Logger) error {
+	var err error
+
+	if config.Debug {
+		if logger, err = zap.NewDevelopment(); err != nil {
+			return err
+		}
+		defer logger.Sync() //nolint
+
+	}
+
+	pm := policy.NewManager()
+	if err := pm.InitializeStore(
+		config.PluginLocations, config.PolicyStoreName, config.PolicyStoreParams,
+	); err != nil {
+		return err
+	}
+	defer pm.Close()
+
+	switch command {
+	case "list":
+		return runListCommand(config, args, pm, logger)
+	case "verify":
+		return runVerifyCommand(config, args, pm, logger)
+	case "add":
+		return runAddCommand(config, args, pm, logger)
+	case "get":
+		return runGetCommand(config, args, pm, logger)
+	case "delete":
+		return runDeleteCommand(config, args, pm, logger)
+	default:
+		return fmt.Errorf("unexpected command: \"%s\"", command)
+	}
+}
+
+func runListCommand(config *common.Config, args []string, pm *policy.Manager, logger *zap.Logger) error {
+	var tenantID int
+
+	listFlags := flag.NewFlagSet("list", flag.ExitOnError)
+	listFlags.IntVar(&tenantID, "t", 0, "List stored policies only for the specified tenant ID.")
+
+	if err := listFlags.Parse(args); err != nil {
+		return err
+	}
+
+	argsRest := listFlags.Args()
+	if len(argsRest) != 0 {
+		return fmt.Errorf("unexpected arguments: %v (see -h for usage)", argsRest)
+	}
+
+	policies, err := pm.ListPolicies(tenantID)
+	if err != nil {
+		return err
+	}
+
+	for _, pentry := range policies {
+		fmt.Println(pentry.TenantID, pentry.TokenFormatName)
+	}
+
+	return nil
+}
+
+func runGetCommand(config *common.Config, args []string, pm *policy.Manager, logger *zap.Logger) error {
+	var tenantID int
+	var outPath string
+
+	addFlags := flag.NewFlagSet("add", flag.ExitOnError)
+	addFlags.IntVar(&tenantID, "t", 1, "TenantID to which the policies should be added.")
+	addFlags.StringVar(&outPath, "o", "policy.zip", "Output zip path.")
+
+	if err := addFlags.Parse(args); err != nil {
+		return err
+	}
+
+	argsRest := addFlags.Args()
+	if len(argsRest) == 0 {
+		return fmt.Errorf("token format(s) not specified (see -h for usage)")
+	}
+
+	var formats []common.TokenFormat
+	for _, fmtName := range argsRest {
+		format, err := common.TokenFormatFromString(fmtName)
+		if err != nil {
+			return err
+		}
+
+		formats = append(formats, format)
+	}
+
+	var policies []*common.Policy
+	for _, format := range formats {
+
+		policy, err := pm.GetPolicy(tenantID, format)
+		if err != nil {
+			return fmt.Errorf(
+				"could not get policy for tenant %d, format %q: %v",
+				tenantID,
+				format.String(),
```

Comment:
here and in other similar instances: if the type is a stringer -- i.e., it provides a `String()` method  -- you can avoid calling it explicitly (`Errorf` will do it for you.)

Reply:
Oh, cool! Thanks!